src/cfnlint/data/ExtendedSpecs/all/03_value_types/aws_rds.json

```
@@ -22,5 +22,13 @@
       "NumberMax": 15,
       "NumberMin": 0
     }
+  },
+  {
+    "op": "add",
+    "path": "/ValueTypes/AWS::RDS::DBInstance.Engine",
+    "value": {
+      "AllowedPattern": "Has to be one of [\"aurora\", \"aurora-mysql\", \"aurora-postgresql\", \"mariadb\", \"mysql\", \"oracle-ee\", \"oracle-se2\", \"oracle-se1\", \"oracle-se\", \"postgres\", \"sqlserver-ee\", \"sqlserver-se\", \"sqlserver-ex\", \"sqlserver-web\"]",
```

These quotation marks get dropped anyway for some reason, so we might as well drop them here so it's a bit cleaner
```suggestion
      "AllowedPattern": "Has to be one of [aurora, aurora-mysql, aurora-postgresql, mariadb, mysql, oracle-ee, oracle-se2, oracle-se1, oracle-se, postgres, sqlserver-ee, sqlserver-se, sqlserver-ex, sqlserver-web]",
```